### ISSUE TITLE
No sound if calling with iOS sdk.

### ISSUE THREAD

**Issue by forses3** [jitsi/jitsi-meet#9193]
### Description:

No sound if calling with iOS SDK.

### Steps to reproduce:

User A starts conference (create a room) with iOS SDK
Another client (User B) receives the VoIP Push and enters the room.

Both users have  microphones turned on and sound of their iOS devices is on.

This bug mostly occurs when iOS device of user B is blocked

### Actual behavior:

No one can hear one another. This result occurs from time to time,

### SDK information:

- Jitsi Meet version: 2.11.0
- Operating System: iOS 14.5.1

### Client information ([ARROUND](https://apps.apple.com/us/app/arround/id1435766899?l=en&ls=1)) :

- App version: 3.4.2
- Operating System: iOS 14.5.1

**Comment by saghul** [comment]
SDK 2.11 is quite outdated. Please try updating to SDK 3 (latest 3.x) and report again. Sorry, we don't have the resources to dig into old releases.

**Comment by saritaOctal** [comment]
Hello,
We are facing issue with version above to iOS 12.0. Device with iOS 12.0 have audio in call but above to iOS 12.0 some times not having audio in Jitsi call with CallKit. But Jitsi call audio voice works without CallKit.
Please let me know what's issue with CallKit.

**Comment by saghul** [comment]
What SDK version are you using?

**Comment by saritaOctal** [comment]
3.0.0

**Comment by saghul** [comment]
Please update to the latest 3.10.x release, we have fixed some sounds issues with iOS 15 (as an example) very recently.

**Comment by saritaOctal** [comment]
Every time sound works when call initiate from iPhone 6 but mostly not works sound when call initial from iPhone X

**Comment by saritaOctal** [comment]
Yes with Native iOS sdk (Pod) its working but when trying with Customized SDK using node not working.

**Comment by saghul** [comment]
Then you need to start looking at the differences between the 2.

**Comment by saritaOctal** [comment]
iOS- The feature of Jitsimeet Call-Integration used in app. App behave weird. 
Like if call initiated(started) from device iPhone 6 , call have voice on both devices.
But when call initiated(started) from iPhone X , both devices haven’t any voice.
